classes/class-twentytwenty-non-latin-languages.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Non-latin language handling.
+ *
+ * Handle non-latin language styles.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty
+ * @since 1.0.0
+ */
+
+if ( ! class_exists( 'TwentyTwenty_Non_Latin_Languages' ) ) {
+	/**
+	 * Language handling.
+	 */
+	class TwentyTwenty_Non_Latin_Languages {
+
+		/**
+		 * Get custom CSS.
+		 *
+		 * Return CSS for non-latin language, if available, or null
+		 * 
+		 * @param string $type Whether to return CSS for the "front-end", "block-editor" or "classic-editor".
+		 *
+		 * @return void
+		 */
+		public static function get_non_latin_css( $type = 'front-end' ) {
+
+			// Fetch users locale.
+			$locale = get_bloginfo( 'language' );
+
+			// Define fallback fonts for non-latin languages.
+			$font_family = apply_filters(
+				'twentytwenty_get_localized_font_family_types',
+				array(
+
+					// Arabic.
+					'ar'    => array( 'Tahoma', 'Arial', 'sans_serif' ),
```

Comment:
Is this meant to be sans-serif? Not sans_serif?

Reply:
Those should all be `sans-serif`. Updated.